### ISSUE TITLE
[APP]https://www.namecheap.com/?gad_source=1&gclid=CjwKCAjwte-vBhBFEiwAQSv_xUPYjcEbVFPMzhAIf8RjwBX66ytuRFYGbqgJiYphRaJaoIk1kkWKoxoCruAQAvD_BwE

### ISSUE THREAD

**Issue by UgnePereviciute** [PipedreamHQ/pipedream#11110]
**Name of app / service**

**Link to developer documentation**

**Is lack of support preventing you from building workflows, or do you have a workaround?**

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**

**Comment by sergio-eliot-rodriguez** [comment]
If they have an API, it should be. I'll check about it. Thanks for the request

**Comment by sergio-eliot-rodriguez** [comment]
Actually, Namecheap is integrated

https://pipedream.com/apps/namecheap

Are there any specific components you need? any trigger and/or action?

**Comment by UgnePereviciute** [comment]
Yes, i found this, but isnt it possible to find namecheap when you trying t add a source? Cause depending on this script i cant connect to the ip. Sorry, I'm new at this, so trying to figure this program out :)

**Comment by UgnePereviciute** [comment]
I think i managed to connect, but what IP will be queried everytime? Cause it changes

**Comment by sergio-eliot-rodriguez** [comment]
IPs are dynamic, Pipedream a workflow is deployed to its own virtual machine in AWS. This means a Pipedream workflow's execution environment has its own RAM and disk, isolated from other users’ workflows. However, outbound traffic shares the same network as other AWS services deployed in the us-east-1 region. That means network requests from Pipedream workflows (e.g. an HTTP request or a connection to a database) originate from the standard range of AWS IP addresses. These AWS, IP range of addresses is documented at: https://docs.aws.amazon.com/vpc/latest/userguide/aws-ip-ranges.html

**Comment by sergio-eliot-rodriguez** [comment]
If you can't find Namecheap via a source is because there aren't Namecheap sources available yet. However you can request one following this link: https://github.com/PipedreamHQ/pipedream/issues/new/choose

and selecting

![image](https://github.com/PipedreamHQ/pipedream/assets/101526248/a9695ef8-8f05-4bc1-8968-5fdabdb085b9)


if there was an action in Namecheap you'd like to see implemented please request it via

![image](https://github.com/PipedreamHQ/pipedream/assets/101526248/4c1e6028-dece-4eb7-9e19-063baf0f0c0b)


I'm closing this ticket because isn't related to an app integration,
